src/vs/workbench/api/common/extHostNotebook.ts

```diff
@@ -395,7 +395,7 @@ export class ExtHostNotebookController implements ExtHostNotebookShape {
 	private _createExtHostEditor(document: ExtHostNotebookDocument, editorId: string, data: INotebookEditorAddData) {
 
 		if (this._editors.has(editorId)) {
-			throw new Error(`editor with id ALREADY EXSIST: ${editorId}`);
+			throw new Error(`editor with id ALREADY EXIST: ${editorId}`);
```

Comment:
**suggestion**: I think this should be "exists", since it's singular.

```suggestion
			throw new Error(`editor with id ALREADY EXISTS: ${editorId}`);
```